person.rb

```diff
@@ -1,29 +1,11 @@
 class Person
-  def initialize(name, age, _parent_permission = 'true')
+  def initialize(age, name = 'Unknown', parent_permission: true)
     @id = id
     @name = name
     @age = age
   end
```

Comment:
- [ ] Here, we have to define `@parent_permission = parent_permission` in initialize, Then only you can use it in the `can_use_service?` method.

Reply:
Man I really dont understand your review. what did  you mean here

Reply:
Here we have to define one instance variable name `parent_permission` also. For ex:-
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
  end
```